cfe_internal/update/update_policy.cf

```diff
@@ -104,21 +155,21 @@ bundle agent cfe_internal_update_policy_cpv
 
     windows::
 
-      "modules_dir"        string => "/var/cfengine/modules",      # NB! NOT $(sys.workdir) on Windows !
+      "modules_dir_source"        string => "/var/cfengine/modules",
```

Comment:
Hum, should we also consider changing these paths to WORKDIR for autotools so that FHS built packages paths would be correct? .

Reply:
Yes, also in quite a few other places. But I'm not trying to fix the world in this PR. Let's leave that for a separate PR.

Reply:
Agreed.

Reply:
there is also a shortcut defined `modules` but do not know in which cfengine version this was introduced.